mec/mec1501/component/qmspi.h

```diff
@@ -143,14 +145,14 @@
 #define MCHP_QMSPI_M_SIG_MODE1_VAL	0x06ul
 #define MCHP_QMSPI_M_SIG_MODE2_VAL	0x01ul
 #define MCHP_QMSPI_M_SIG_MODE3_VAL	0x07ul
-#define MCHP_QMSPI_M_SIG_MODE0		(0x00ul << MCHP_QMSPI_M_SIG_POS)
+#define MCHP_QMSPI_M_SIG_MODE0		0x00ul
 #define MCHP_QMSPI_M_SIG_MODE1		(0x06ul << MCHP_QMSPI_M_SIG_POS)
 #define MCHP_QMSPI_M_SIG_MODE2		(0x01ul << MCHP_QMSPI_M_SIG_POS)
 #define MCHP_QMSPI_M_SIG_MODE3		(0x07ul << MCHP_QMSPI_M_SIG_POS)
 #define MCHP_QMSPI_M_CS_POS		12u
 #define MCHP_QMSPI_M_CS_MASK0		0x03ul
 #define MCHP_QMSPI_M_CS_MASK		(0x03ul << 12)
-#define MCHP_QMSPI_M_CS0		(0x00ul << 12)
+#define MCHP_QMSPI_M_CS0		0x00ul
```

Comment:
@scottwcpg could you confirm this is the latest value, in the original espi_qmspi_update branch shared with SAF driver this was 
#define MCHP_QMSPI_M_CS0		(0x00ul << 12)

Reply:
Hi @albertofloyd 
I was trying out one of the linting tools in neovim and it complained that shifting the value 0 was was redundant.
I will put it back for clarity in next rebase.